.github/workflows/build_deploy_pages.yml

```diff
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build docs and deploy to pages
+
+on:
+  push:
+    branches: [main, pages]
```

Comment:
Do we need a pages branch?

Reply:
Ah, no we don't. That's a relic from getting things to work on my fork without pushing into main. I'll remove